bigquery/storage/managedwriter/appendresult.go

```diff
@@ -22,6 +22,10 @@ import (
 	"google.golang.org/protobuf/types/known/wrapperspb"
 )
 
+// NoOffset is a sentinel value for signalling we're not tracking
+// stream offset (e.g. a default stream which allows simultaneous append streams).
+const NoOffset int64 = -1
```

Comment:
nit: maybe NoStreamOffset. As the package grows I could see this being harder to identify what it is for.

Reply:
Done.